Sources/AblyAssetTrackingPublisher/PublisherTrackableState.swift

```diff
@@ -0,0 +1,44 @@
+ //
```

Comment:
Just like Quintin mentioned (maybe in an unrelated PR), I think we should remove the Xcode autogenerated comments at the top, since they are not very useful. I remember at one place I worked, a developer mentioned that it was nice/ gave them pride to have their name at the top. I hope removing this autogenerated comment doesn't negatively affect morale 😅

Reply:
8ff3ce15045fea1dcc47643f5e3c093af7984976

Reply:
I guess this wasn't committed, I can't find this commit: https://github.com/ably/ably-asset-tracking-swift/pull/181/commits

Reply:
8ff3ce15045fea1dcc47643f5e3c093af7984976

Reply:
if it's ok to you, please resolve conversation and we can merge this one